shakedown/dcos/__init__.py

```diff
@@ -162,3 +260,86 @@ def _gen_url(url_path):
     """
     from six.moves import urllib
     return urllib.parse.urljoin(dcos_url(), url_path)
+
+
+# Initialize some things.
```

Comment:
comment doesn't make sense... creating a new object "initializes some things".   Is this comment necessary?

Reply:
I will update it to reference what it is actually doing - creating a `TransportManager`.